quinn/src/connection.rs

```diff
@@ -857,16 +857,17 @@ impl State {
         let mut transmits = 0;
 
         let max_datagrams = self.socket.max_transmit_segments();
-
-        while let Some(t) = self.inner.poll_transmit(now, max_datagrams) {
+        let mut buffer = BytesMut::with_capacity(1500);
```

Comment:
Use MTU.

Reply:
I need a pointer regarding this... I've investigated but I'm could not understand from where should I get the MTU value in this case.